rdr_service/dao/questionnaire_response_dao.py

```diff
@@ -948,10 +948,10 @@ def _update_participant_summary(
                 participant_summary.idVerificationOrigin = IdVerificationOriginType.REMOTE
                 if not participant_summary.firstIdVerifiedOn:
                     participant_summary.firstIdVerifiedOn = datetime.utcfromtimestamp(remote_id_info['verified_on'])
```

Comment:
Do you have to confirm first now that the `verified_on` field has a value before you try to use it as a timestamp, now that it's not checked on line 942?    

If there's a chance the value could ever be missing (even if that's not expected), I think line 950 would throw an  exception in the `utcfromtimestamp` call.   Also, isn't `firstIdVerifiedOn` a Date field (vs. Datetime) in the model?

Reply:
thanks Peggy, I have added the code for the verified_on part of this review. 

yes, firstIdVerifiedOn is a date field in the model. This is using the datetime.datetime python library to get the utctimestamp, and then saving as a Date in the db.